pkg/analysis/nomaps/testdata/src/a/a.go

```diff
@@ -0,0 +1,66 @@
+package a
+
+type (
+	MapStringComponent    map[string]Component
+	PtrMapStringComponent *map[string]Component
+	MapStringInt          map[string]int
+	MapIntString          map[int]string
```

Comment:
In other type check linters, we also linting the type declarations, should we be consistent here?

Reply:
I think linting the type type declarations are not necessary, because when we fix the struct field, it means we don't have the specific type in CRD.
When we have type declarations defined that contain map[K]V or map[string]string types, these types may be used in files other than CRDs and it's no problem.
If you want to check type declarations as well, I think it's better to use it only when the configuration is Enforce. What do you think?